pytaskflow/storage/memory_storage.py

```
@@ -106,8 +111,16 @@ def set_job_state(self, job_id: str, state: BaseState, expected_old_state: Optio
             
             job.state_name = state.name
             job.state_data = state.serialize_data()
+            if state.name == EnqueuedState.NAME:
+                self._condition.notify() # Notify if job is re-enqueued
```

Flake8 (7.2.0)

[error] at least two spaces before inline comment

(E261)